bad.py

```
@@ -0,0 +1,183 @@
+# DO NOT USE THIS IN PRODUCTION. This file intentionally contains many anti-patterns, style violations,
+# insecure practices, and bad code smells for testing review tools.
+
+import os, sys, logging, shutil, json, random
+from math import *   # wildcard import (bad)
+# duplicate and unused imports
+import urllib.request
+import urllib.parse
+import time as time_module
+
+# Hardcoded secret (intentional bad practice)
+HF_TOKEN = "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"
```

Ruff (0.13.1)

Possible hardcoded password assigned to: "HF_TOKEN"

(S105)